src/creatwth.cpp

```diff
@@ -714,6 +728,10 @@ BOOL WINAPI DetourUpdateProcessWithDllEx(_In_ HANDLE hProcess,
         DETOUR_CLR_HEADER clr;
         CopyMemory(&clr, &der.clr, sizeof(clr));
         clr.Flags &= ~COMIMAGE_FLAGS_ILONLY;    // Clear the IL_ONLY flag.
+        //The above sentence clears the IL_ONLY flag, 
+        //so we need to restore the IL_ONLY flag after the IAT is restored after starting the process completion 
```

Comment:
>// [](start = 8, length = 2)

nit: all comments should have an empty space after comment characters, that's the style.

Reply:
The latest commit has resolved it.